pkg/kn/commands/test_helper.go

```diff
@@ -67,3 +69,26 @@ Eventing: Manage event subscriptions and channels. Connect up event sources.`,
 	flag.CommandLine.Parse([]string{})
 	return rootCmd
 }
+
+func TestContains(t *testing.T, output string, sub []string, element string) {
+	for _, each := range sub {
+		if !strings.Contains(output, each) {
+			t.Errorf("Missing %s: %s", element, each)
+		}
+	}
+}
+
+func AreSlicesEqual(a, b []string) bool {
```

Comment:
this seems like could just be replaced with `reflect.DeepEqual`

Reply:
thanks, updated